examples/resources/powerscale_subnet/resource.tf

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# Available actions: Create, Update, Delete and Import
+# After `terraform apply` of this example file it will create a new subnet with the name set in `name` attribute on the PowerScale
+
+# PowerScale subnet allows you to manage the subnet on the Powerscale array
+resource "powerscale_subnet" "subnet" {
```

Comment:
Hi @NavisJ ,could you add optional fields for creating and updating in the example file as well?